requirements-ts.txt

```diff
@@ -1,3 +1,4 @@
 sktime==0.5.3
 tbats==1.1.0
 pmdarima>=1.8.0
+fbprophet>=0.7.1
```

Comment:
Would bats, tbats be optional dependencies eventually?
What about pmdarima and prophet?

Reply:
yes bats, tbats, fbprophet can be optional dependencies, in that case, it catches an `ImportError`.